.github/fabricbot.json

```diff
@@ -5,31 +5,751 @@
 		"subCapability": "IssuesOnlyResponder",
 		"version": "1.0",
 		"config": {
-			"conditions": {
-				"operator": "and",
+		"conditions": {
+			"operator": "and",
+			"operands": [
+			{
+				"name": "labelAdded",
+				"parameters": {
+				"label": "need-info"
+				}
+			}
+			]
+		},
+		"eventType": "issue",
+		"eventNames": [
+			"issues",
+			"project_card"
+		],
+		"taskName": "Add comment when 'need-info' is applied to issue",
+		"actions": [
+			{
+			"name": "addReply",
+			"parameters": {
+				"comment": "Hi @${issueAuthor}. We have added the \"need-info\" label to this issue, which indicates that we have an open question for you before we can take further action. This issue will be closed automatically in 7 days if we do not hear back from you by then - please feel free to re-open it if you come back to this issue after that time."
+			}
+			}
+		]
+		}
+	},
+	{
+		"taskType": "scheduled",
+		"capabilityId": "ScheduledSearch",
+		"subCapability": "ScheduledSearch",
+		"version": "1.1",
+		"config": {
+		"frequency": [
+			{
+			"weekDay": 1,
+			"hours": [
```

Comment:
We can write this in a single like, rather than using 24 lines just to say per hour.
```js
"hours": [0,1,2,... ]
```